src/makeCalculator.js

```diff
@@ -36,8 +36,36 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
```

Comment:
Add spaces to improve the code readability 
```suggestion
    },
    
    subtract(a, b) {
```